model_templates/training/r_lang/create_pipeline.R

```diff
@@ -0,0 +1,21 @@
+create_pipeline<-function(X, y) {
```

Comment:
This is a lint comment, but traditionally you put spaces around operators like `<-`.

E.g. `create_pipeline <- function(X, y) {`